pom.xml

```diff
@@ -49,6 +49,7 @@
 		<sonar.coverage.jacoco.xmlReportPaths>${project.reporting.outputDirectory}/target/jacoco-report/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
 		<sonar.organization>catenax-ng</sonar.organization>
 		<sonar.projectKey>catenax-ng_tx-vas-country-risk-backend</sonar.projectKey>
+		<sonar.projectName>tx-vas-country-risk-backend</sonar.projectName>
```

Comment:
Please prioritize eclipse-tractusx as leading repo. Therefore the naming should also follow Tractus-X. So maybe do not add the `tx-` prefix. You can add conditions to your workflows, that will configure the `sonar.projectName` via command parameter